Userland/Libraries/LibCore/System.cpp

```diff
@@ -997,6 +997,16 @@ ErrorOr<int> mkstemp(Span<char> pattern)
     return fd;
 }
 
+ErrorOr<char*> mkdtemp(Span<char> pattern)
```

Comment:
The whole point of these wrappers is to C++-ify them, and that includes not taking or returning `char*`s  - see `readlink()` for an example (except this new function should use the fallible `String`)